### ISSUE TITLE
Check the usage of parentheses in names of Chinese variants

### ISSUE THREAD

**Issue by amire80** [wikimedia/language-data#141]
Many names of Chinese variants here have parentheses. Here's the full current list:
```
cjy-hans: [Hans, [AS], 晋语（简化字)]
gan-hans: [Hans, [AS], 赣语（简体）]
hak-hans: [Hans, [AS], 客家语（简体）]
hak-hant: [Hant, [AS], 客家語（繁體）]
zh-cn: [Hans, [AS], 中文（中国大陆）]
zh-hans: [Hans, [AS], 中文（简体）]
zh-hant: [Hant, [AS], 中文（繁體）]
zh-hk: [Hant, [AS], 中文（香港）]
zh-mo: [Hant, [AS], 中文（澳門）]
zh-my: [Hans, [AS], 中文（马来西亚）]
zh-sg: [Hans, [AS], 中文（新加坡）]
zh-tw: [Hant, [AS], 中文（台灣）]
```

One small issue is in cjy-hans: it has ASCII closing parentheses, while all the rest have fullwidth parentheses on both sides.

But a bigger issue is that I wonder whether the parentheses are needed at all. Can these names be written without parentheses? Or is it more natural with parentheses?

Unfortunately I don't know Chinese. For what it's worth, cdo-hani has no parentheses ("閩東語漢字"), and I can find some others without parentheses, e.g. "客家語繁體" (even if I remove Wikipedia and translatewiki from the results).

**Comment by yfdyh000** [comment]
Using parentheses is a neutral and uniform approach to them.
A few variants have common non-bracket writing, such as zh-cn: 简体中文, zh-tw: 繁体中文 (or 正体中文), but this is not universal for these variants. Non-bracketed titles are feasible and once used but crappy (like "大陆简体", etc).
The full-width brackets are because zhwiki praises it and no-space spacing. This is not a significant problem. In non-zhwiki, spacing + half-width punctuation may be more common to build the languages list.

**Comment by amire80** [comment]
Excellent, thanks for the clarification!

**Comment by amire80** [comment]
Resolved in #148. Thanks for the comments and the reviews @Abijeet @kartikm @yfdyh000 !
